common.lic

```diff
@@ -282,4 +282,27 @@ module DRC
       .map { |_name, properties| properties['abbrev'] }
       .each { |abbrev| fput("release #{abbrev}") }
   end
+
+  def necro_harvest_check
+    if !@necro_store
```

Comment:
I added this and then had to remove the use of the rummage helper, as it returns an input that errors out with the .scan usage.